### PR TITLE
Add code Linter

### PR DESCRIPTION
**Pull request by neelshah2409**
Implemented Pint Linter and linting the existing code base .

### CODE REVIEW

File: data/data.php [hunk 1]
```diff
@@ -51453,6 +51453,14 @@
...
 		],
 	],
+	'*.user.fm' => [
+		'suffix' => '*.user.fm',
+		'type' => 'PRIVATE',
+		'comments' => [
+			'Fastmail : https://www.fastmail.com/',
+			'Submitted by Marc Bradshaw <marc@fastmailteam.com>',
+		],
+	],
```
Comment (Meldiron) [inline data/data.php]:
```suggestion
```

Can we revert this change? Not sure how it got here, but it's not relevant to linter. I think we can safely remove it.

Comment (neelshah2409) [reply data/data.php]: Hey @Meldiron Could you help me inreverting this as I am not sure if it could be safely removed.

**Comment by neelshah2409** [comment]
@Meldiron I didn't knew what happened when I tried to revoke the last commit ,I would make another PR for this .
And really sorry for this .
